### PR TITLE
BaSP-TG-123: Logical Recovery and Final Removal

### PR DESCRIPTION
**Pull request by Javier-Cervera**
Endpoints for restoration or final elimination of Admin/Employee/Projects/Tasks/Timesheets

### CODE REVIEW

File: src/controllers/tasks.js [hunk 1]
```diff
@@ -141,10 +141,44 @@ const deleteTask = async (req, res) => {
...
+    if (!result) {
+      return res.status(404).json({
+        message: 'Task not found',
+        data: undefined,
+        error: true,
+      });
+    }
+    return res
+      .json({
+        message: 'Task succesfully removed',
```
Comment (iaracriscenti) [inline src/controllers/tasks.js:163]: Missing a s in 'succesfully', it sould be 'successfully'

File: src/controllers/time-sheets.js [hunk 2]
```diff
@@ -147,10 +157,41 @@ const deleteTimesheet = async (req, res) => {
...
+    const result = await models.TimeSheet.findByIdAndDelete(req.params.id);
+    if (!result) {
+      return res.status(404).json({
+        message: `There is no timesheet with this Id ${req.params.id}`,
+        data: undefined,
+        error: true,
+      });
+    }
+    return res.status(200).json({
+      message: `The ${req.params.id} timesheet has been susccesfully removed`,
```
Comment (iaracriscenti) [inline src/controllers/time-sheets.js:178]: Change 'susccesfully' for 'successfully'

**Review by Johsua30 — APPROVED** [review]
Everything in order

**Review by iaracriscenti — APPROVED** [review]
Locally tested with FE TG-123 and it works!

**Pull request merged**